doc/reporters/github-reporter.md

```diff
@@ -2,6 +2,20 @@
 
 Report about design and static code diagnostics issues in pull requests based on GitHub Actions Workflow commands.
 
+1. Your git workflow must include/support flutter/dart
```

Comment:
It might be not so clear for the external user what does this phrase mean. Seems like they might expect an example here.